src/components/menu/menu-full-screen/menu-full-screen.component.js

```diff
@@ -47,6 +47,20 @@ const MenuFullscreen = ({
     }
   }, [isOpen]);
 
+  const scrollVariants = {
+    light: "light",
+    dark: "dark",
+    white: "light",
+    black: "dark",
+  };
+
+  const iconColors = {
```

Comment:
May I ask what the reasoning was for defining the colours here with the component? Is this information just needed for `Menu` or could this potentially be abstracted to the theme if other components need use this?